src/main/java/javaserver/ServerFactory.java

```diff
@@ -9,9 +9,9 @@
 import java.net.Socket;
 
 public class ServerFactory {
+
 	public Server createServer(int port) throws IOException {
```

Comment:
What you could do instead is move all of these methods into their respective classes, make their constructors `private`, and create a `public static` method that creates the objects.